hotsos/core/host_helpers/cli/catalog.py

```diff
@@ -713,6 +733,18 @@ def __init__(self):
             'ip_link':
                 [BinCmd('ip -s -d link'),
                  FileCmd('sos_commands/networking/ip_-s_-d_link')],
+            'kubectl_logs':
+                [KubectlLogsBinCmd('kubectl'),
+                 # Canonical K8s
+                 KubectlLogsBinCmd('microk8s.kubectl'),
+                 # MicroK8s
```

Comment:
My only comment is that the comments don't seem to match the commands?

Reply:
Oops! Thanks Brian will fix.